src/navigators/nameScreen.ts

```diff
@@ -0,0 +1,9 @@
+export enum nameScreen {
```

Comment:
enum should starts from Upper case letter

Reply:
got it, i will fixed